vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -226,6 +229,107 @@
         server foia0 {{ internal_host }}:488 check fall 3 rise 2 check ssl verify required ca-file {{ ssl_ca }}
   when: foia_portal_context is defined
 
+- name: Set up proxies to microservices
+  block:
+    - name: Populate haproxy.conf with fronted part 
+      become: yes
+      insertafter: "#arkcase.next services"
+      path: "/etc/opt/rh/rh-haproxy18/haproxy/haproxy.cfg"
+      block: |
+        acl acl_schema-registry path_beg /schema-registry
+        acl acl_elasticsearch path_beg /elasticsearch
+        acl acl_gateway path_beg /gateway
+        acl acl_eureka path_beg /eureka
+        acl acl_eureka-dashboard path_beg /eureka-dashboard
+        acl acl_control-center path_beg /control-center
+        acl acl_zipkin-health path_beg /zipkin/health
+        acl acl_zipkin-api path_beg /zipkin/api
+        acl acl_zipkin path_beg /zipkin
+        acl acl_config path_beg /config
+    - name: Tell fronted part which backend servers will use
+      become: yes
+      insertafter: "# backend uses"
+      path: "/etc/opt/rh/rh-haproxy18/haproxy/haproxy.cfg"
+      block: |
```

Comment:
Best to use a specific `marker` attribute in all block elements.

Reply:
Yes, i will include marker, thank you